recourse_methods/test_mrmc_method.py

```diff
@@ -130,8 +130,11 @@ def test_cluster_data(self, mock_kmeans):
         expected_cluster_centers = "mock centers"
         mock_kmeans().fit_predict.return_value = expected_cluster_assignments
         mock_kmeans().cluster_centers_ = expected_cluster_centers
+        seed = 199318
```

Comment:
Nit: Throughout this PR request, you use `random_seed` and `seed` -- please update to `random_seed` for consistency (or use one or the other).